test/tests/outputs/format/test_json.py

```diff
@@ -11,8 +11,8 @@ def find_app():
     """
     moose_dir = os.environ.get("MOOSE_DIR")
     if not moose_dir:
-        file_dir = os.dirname(os.path.abspath(__file__))
-        moose_dir = os.dirname(os.dirname(file_dir))
+        file_dir = os.path.dirname(os.path.abspath(__file__))
+        moose_dir = os.path.abspath(os.path.join(file_dir, "..", "..", "..", ".."))
```

Comment:
To use this I would need to find the top level moose directory first, which is what this is doing. Are you suggesting replacing the entire `find_app()` function?